virt-v2v/cold/entrypoint.go

```diff
@@ -86,6 +87,12 @@ func main() {
 			}
 		}
 		virtV2vArgs = append(virtV2vArgs, "-ip", "/etc/secret/secretKey")
+		macsToIps := os.Getenv("V2V_staticIPs")
```

Comment:
Since we only appending this var to the pod only when its its not empty, you can use `checkEnvVariablesSet(V2V_staticIPs)` for the IF check,  and then get its value directly in the for loop.